cdoc/cdoc_web/src/main/java/com/cdocweb/App.java

```diff
@@ -0,0 +1,75 @@
+package com.cdocweb;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.io.File;
+import java.io.FileWriter;
+
+
+import static spark.Spark.get;
+import static spark.Spark.port;
+import static spark.Spark.post;
+
+public class App 
+{
+    public static void main( String[] args )
+    {
+        port(4444);
```

Comment:
Interesting port choice, why not 80 and/or why not take value from env?

Reply:
Security reason, port 80 is prone to automated scans